### ISSUE TITLE
Support individual filepaths in `scan --path`

### ISSUE THREAD

**Issue by JodanJodan** [JohnDoee/autotorrent2#56]
I've been using this post-completion hook in qBittorrent:

`~/bin/at2 scan -p "%F"`

`"%F"` is the content path, so the root folder for multifile torrents or just the file path for single file torrents. I believe this used to work, but recently after updating I get the following error:

`Failed to scan /~/files/[Group] Release - 01.mkv: [Errno 20] Not a directory: '/~/files/[Group] Release - 01.mkv'`

Could single file paths be supported (again?)?

**Comment by JohnDoee** [comment]
can you try downgrading and see if that fixes it?

> pip install autotorrent2==1.2.3

**Comment by JodanJodan** [comment]
Yeah I think I was just mistaken about it being a regression and I just never ran into it as most of my torrents were multifile. It'd be a nice addition though, instead of having to rescan the whole directory.

**Comment by JohnDoee** [comment]
It will be out with next release, made a typo in commit message but it is fixed in the latest commit.
